sklearn/preprocessing/label.py

```diff
@@ -108,25 +108,9 @@ def fit(self, y):
         y = column_or_1d(y, warn=True)
         _check_numpy_unicode_bug(y)
         self.classes_ = np.unique(y)
+        self._hashtable = {c: i for i, c in enumerate(self.classes_)}
```

Comment:
hashtable describes the datastructure, not the semantics. Call it `_class_lookup` or similar.